### ISSUE TITLE
Expose error definitions

### ISSUE THREAD

**Issue by at-wat** [pion/sctp#251]
`webrtc.DataChannel.OnError(err error)` often receives 
- `abort chunk, with following errors: (User Initiated Abort: Close called)`.
- `sending reset packet in non-established state: state=Closed`

which seems be intended results. (Maybe by the recent updates?)

### Possible solutions
- Expose base errors (`var errErrorName` -> `var ErrErrorName`) for `errors.Is()`
- Define struct for errors with values for `errors.As()`
  e.g. `errChunk`,
  ```go
  type ErrChunk struct {
    Name string
    Causes []string
  }
  func (e *ErrChunk) Error() {
    var errStr string
    for _, e := range e.Causes {
      errStr += fmt.Sprintf("(%s)", e)
    }
    return fmt.Sprintf("[%s] abort chunk, with following errors: %s", e.Name, errStr)
  }

**Comment by at-wat** [comment]
> `webrtc.DataChannel.OnError(err error)` often receives
>  * `abort chunk, with following errors: (User Initiated Abort: Close called)`.
>  * `sending reset packet in non-established state: state=Closed`
> 
> which seems be intended results. (Maybe by the recent updates?)

#183 seems related

**Comment by edaniels** [comment]
We're seeing these errors (specifically `sending reset packet in non-established state` plenty of the time now and are logging them as such. Is there a way to quiet these if they are normal operating errors?

**Comment by at-wat** [comment]
@jeremija any idea about above error?

**Comment by jeremija** [comment]
This is most likely as side effect of https://github.com/pion/sctp/pull/183

I'm not sure if we should skip the callback or expose the errors, but exposing the errors make sense to me as it's a smaller change.

**Comment by at-wat** [comment]
Released v1.8.6

**Comment by at-wat** [comment]
Using exposed errors in my product works as expected now
